src/Raven.Client/Documents/Subscriptions/SubscriptionWorker.cs

```diff
@@ -561,7 +563,7 @@ private async Task ProcessSubscriptionAsync()
                             throw;
                         }
 
-                        notifiedSubscriber = Task.Run(async () => // the 2'nd thread
+                        notifiedSubscriber = _notifiedSubscriber = Task.Run(async () => // the 2'nd thread
```

Comment:
is there a momet when notifiedSubscriber != _notifiedSubscriber ?
I'm wondering why we need a field at all? because it looks like you could define this local variable (notifiedSubscriber) in outer scope, before `tryf` ?

Reply:
done